api/build.gradle

```diff
@@ -0,0 +1,130 @@
+
+plugins {
+    id 'antlr'
+    id 'checkstyle'
+    alias(libs.plugins.spring.boot)
+    alias(libs.plugins.git.properties)
+    alias(libs.plugins.jib)
+}
+
+dependencies {
+    implementation project(":contract")
+    if (prod) {
+        implementation project(":frontend")
+    }
+    implementation project(":serde-api")
+    implementation libs.spring.starter.webflux
+    implementation libs.spring.starter.security
+    implementation libs.spring.starter.actuator
+    implementation libs.spring.starter.logging
+    implementation libs.spring.starter.oauth2.client
+    implementation libs.spring.boot.actuator
+    compileOnly libs.spring.boot.devtools
+
+    implementation libs.spring.security.ldap
+
+    implementation libs.kafka.clients
+
+    implementation libs.apache.avro
+    implementation libs.apache.commons
+    implementation libs.apache.commons.pool2
+    implementation libs.apache.datasketches
+
+    implementation libs.confluent.schema.registry.client
+    implementation libs.confluent.avro.serializer
+    implementation libs.confluent.protobuf.serializer
+    implementation libs.confluent.json.schema.serializer
+
+    implementation libs.aws.msk.auth
+    implementation (libs.azure.identity) {
+        exclude group: 'io.netty', module: 'netty-tcnative-boringssl-static'
```

Comment:
Let's please explain why we are excluding this

Reply:
Same as in maven